sentinel-adapter/sentinel-reactor-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/reactor/MonoSentinelOperatorIntegrationTest.java

```diff
@@ -73,6 +73,12 @@ public void testFluxToMonoNextThenCancelSuccess() {
     @Test
     public void testEmitSingleLongTimeRt() {
         String resourceName = createResourceName("testEmitSingleLongTimeRt");
+        // Warmup once
```

Comment:
Could you please illustrate why we need a warmup here in the code comment?

Reply:
done